homework/homework_part_2/js/app.js

```diff
@@ -0,0 +1,75 @@
+// 1. The difference between interpolation and concatenation is the syntax.
+// 	  Interpolation looks like this: console.log(`This is how ${interpolation} works')
+// 	  Concatenation looks like this: console.log('We ' + 'can ' + concatenate ' like this!')
+
+// 2. DRY stands for Don't Repeat Yourself. You should pay attention to it because
+// 	  it directly effects your code and how efficiently you write it. We can use
+// 	  while and for loops to make sure that we are writing DRY code.
+
+// 3. Declaring a variable tells the compiler that there is a variable. Defining it
+// 	  gives the variable a value.
+
+// 4. Always use const instead of let unless the system yells at you to do otherwise
+
+// 5. A parent directory is a folder that houses other files or folders
+
+// 6. Use a man page by typing man <the thing you need help with> or use help
+
+// 7. Tab completion is the best tool ever because you cut key strokes by allowing
+//	  the terminal window to auto complete stuff for you based what is in the folder
+
+// a < b;
+// c > d;
+// 'Name' === 'Name';
+// a + b = c;
+// a * a = d;
+// e === 'Kevin';
+// 48 == '48'; 
+
+// Loop 1:
+// The while loop is an infinite loop because it does not have a condition listed that will
+// make it stop running.
+
+// Loop 2:
+// This is an infinite loop because you cannot redeclare a const. So you will get an error message.
```

Comment:
Nice! Chris ain't gonna be tricked by a trick question, it _would be_ infinite if error message didn't stop it